runtime/moonbase/src/lib.rs

```diff
@@ -1641,145 +1641,150 @@ mod benches {
-moonbeam_runtime_common::impl_runtime_apis_plus_common! {
-	impl sp_transaction_pool::runtime_api::TaggedTransactionQueue<Block> for Runtime {
-		fn validate_transaction(
-			source: TransactionSource,
-			xt: <Block as BlockT>::Extrinsic,
-			block_hash: <Block as BlockT>::Hash,
-		) -> TransactionValidity {
-			// Filtered calls should not enter the tx pool as they'll fail if inserted.
-			// If this call is not allowed, we return early.
-			if !<Runtime as frame_system::Config>::BaseCallFilter::contains(&xt.0.function) {
-				return InvalidTransaction::Call.into();
-			}
-
-			// This runtime uses Substrate's pallet transaction payment. This
-			// makes the chain feel like a standard Substrate chain when submitting
-			// frame transactions and using Substrate ecosystem tools. It has the downside that
-			// transaction are not prioritized by gas_price. The following code reprioritizes
-			// transactions to overcome this.
-			//
-			// A more elegant, ethereum-first solution is
-			// a pallet that replaces pallet transaction payment, and allows users
-			// to directly specify a gas price rather than computing an effective one.
-			// #HopefullySomeday
-
-			// First we pass the transactions to the standard FRAME executive. This calculates all the
-			// necessary tags, longevity and other properties that we will leave unchanged.
-			// This also assigns some priority that we don't care about and will overwrite next.
-			let mut intermediate_valid = Executive::validate_transaction(source, xt.clone(), block_hash)?;
-
-			let dispatch_info = xt.get_dispatch_info();
-
-			// If this is a pallet ethereum transaction, then its priority is already set
-			// according to effective priority fee from pallet ethereum. If it is any other kind of
-			// transaction, we modify its priority. The goal is to arrive at a similar metric used
-			// by pallet ethereum, which means we derive a fee-per-gas from the txn's tip and
-			// weight.
-			Ok(match &xt.0.function {
-				RuntimeCall::Ethereum(transact { .. }) => intermediate_valid,
-				_ if dispatch_info.class != DispatchClass::Normal => intermediate_valid,
-				_ => {
-					let tip = match &xt.0.preamble {
-						Preamble::Bare(_) => 0,
-						Preamble::Signed(_, _, signed_extra) => {
-							// Yuck, this depends on the index of ChargeTransactionPayment in SignedExtra
-							let charge_transaction_payment = &signed_extra.7;
-							charge_transaction_payment.tip()
-						},
-						Preamble::General(_, _) => 0,
-					};
-
-					let effective_gas =
-						<Runtime as pallet_evm::Config>::GasWeightMapping::weight_to_gas(
-							dispatch_info.total_weight()
-						);
-					let tip_per_gas = if effective_gas > 0 {
-						tip.saturating_div(effective_gas as u128)
-					} else {
-						0
-					};
-
-					// Overwrite the original prioritization with this ethereum one
-					intermediate_valid.priority = tip_per_gas as u64;
-					intermediate_valid
+moonbeam_runtime_common::impl_runtime_apis_plus_common!(
```

Comment:
Can you avoid to change the indentation, it's very hard for reviewers to check in some code changes or not
You can do something like that to keep the same indentation even with the new macro param:

macro_name!({
 ...
 },{
 });

Reply:
I prefer to keep the format, for reviewing this part, you can copy and paste it locally and see the diff.